src/components/form/components/__test__/select.test.js

```diff
@@ -47,3 +47,20 @@ describe('Select component', () => {
     expect(component.toJSON()).toMatchSnapshot();
   });
 });
+
+describe('Select value PropTypes', () => {
+  it('Should be multioption and should not accept non array value', () => {
+    const spy = jest.spyOn(console, 'error').mockImplementation();
+    const component = renderer.create(<Select multiple value={1}></Select>);
+    expect(component.toJSON()).toMatchSnapshot();
+    expect(spy).toBeCalled();
+    spy.mockRestore();
+  });
+  it('Should not be multioption and should not accept array value', () => {
+    const spy = jest.spyOn(console, 'error').mockImplementation();
+    const component = renderer.create(<Select value={[1, 2, 3]}></Select>);
+    expect(component.toJSON()).toMatchSnapshot();
+    expect(spy).toBeCalled();
+    spy.mockRestore();
+  });
+});
```

Comment:
Can you move this `describe` block inside the `'Select component'` describe block?